protocol/src/main/java/org/fido/iot/protocol/To0ServerService.java

```diff
@@ -25,7 +30,10 @@ protected void doHello(Composite request, Composite reply) {
     getStorage().started(request, reply);
   }
 
-  protected void doOwnerSign(Composite request, Composite reply) {
+  protected void doOwnerSign(Composite request, Composite reply)
+      throws NoSuchAlgorithmException, InvalidKeyException {
+    getStorage().setOvKeysAllowlist();
```

Comment:
Move both DB import operations to RvsContextListener, so that this is done only once upon application's startup.

Reply:
Done